### PR TITLE
Config updates

### PR DESCRIPTION
**Pull request by rikvdh**

### CODE REVIEW

**Comment by rikvdh** [comment]
Build is fine, Travis horks on the coveralls thing.

File: config.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 Jerry Jacobs. All rights reserved.
+// Copyright 2017 Jerry Jacobs & Rik van der Heijden. All rights reserved.
```
Comment (xor-gate) [inline config.go]: maybe debpkg authors, with a AUTHORS file?

File: config_test.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
+package debpkg
```
Comment (xor-gate) [inline config_test.go]: add copyright header

**Comment by xor-gate** [comment]
LGTM, updating the year of copyright is not necessary see:

https://golang.org/doc/contribute.html#copyright

> Files in the repository are copyright the year they are added. Do not update the copyright year on files that you change.

**Comment by xor-gate** [comment]
https://github.com/golang/go/blob/master/LICENSE#L1

**Comment by xor-gate** [comment]
Maybe we would like the same style copyright header and LICENSE file first line as the go project with this PR

**Pull request merged**